ios-whitelist.json

```
@@ -277,6 +277,10 @@
     {
       "name": "RxSwift",
       "version": "4.4.1"
+    },
+    {
+      "name": "MLFirebase",
+      "version": null
```

@jbrunoAtMeli Hay que fijar el major version:
```suggestion
      "version": "^~>\\s?0.[0-9]+"
```